packages/vuetify/src/styles/settings/_utilities.scss

```diff
@@ -312,6 +312,16 @@ $utilities: map-deep-merge(
       class: text,
       values: left right center justify start end
     ),
+    "text-decoration": (
+      property: text-decoration,
+      class: text-decoration,
+      values: (
+        normal: none,
```

Comment:
so now i'm not sure if it should be `text-decoration-none` here, it will match the prop value but still may be confused with `text-none` (the one without infix), thoughts?

Reply:
I'll just add both.